### PR TITLE
removed guvnor and jbpm-form-modeler from jobs

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: job-dsls/jobs/deploy_jobs.groovy [hunk 1]
```diff
@@ -92,10 +92,6 @@ def final REPO_CONFIGS = [
                 ircNotificationChannels: ["#guvnordev"],
                 downstreamRepos        : ["guvnor"]
```
Comment (psiroky) [inline job-dsls/jobs/deploy_jobs.groovy]: Please adjust also the downstream repos.

**Comment by mbiarnes** [comment]
@psiroky now good?

File: job-dsls/jobs/deploy_jobs.groovy [hunk 2]
```diff
@@ -90,11 +90,7 @@ def final REPO_CONFIGS = [
         "kie-uberfire-extensions"   : [
                 timeoutMins            : 40,
                 ircNotificationChannels: ["#guvnordev"],
-                downstreamRepos        : ["guvnor"]
-        ],
-        "guvnor"                    : [
-                ircNotificationChannels: ["#guvnordev"],
-                downstreamRepos        : ["kie-wb-playground"]
+                downstreamRepos        : []
```
Comment (psiroky) [inline job-dsls/jobs/deploy_jobs.groovy]: Sorry, should have been more clear. This needs to be `downstreamRespo: [kie-wb-playground]`.

File: job-dsls/jobs/deploy_jobs.groovy [hunk 3]
```diff
@@ -105,10 +101,6 @@ def final REPO_CONFIGS = [
                 ircNotificationChannels: ["#guvnordev"],
                 downstreamRepos        : ["jbpm-form-modeler"]
```
Comment (psiroky) [inline job-dsls/jobs/deploy_jobs.groovy]: This needs to be replaced with `downstreamRepos        : ["drools-wb"]`

**Pull request merged**